### PR TITLE
resnet18 img detection

### LINKED ISSUE #18: write resnet18 section

### PR DESCRIPTION
**Pull request by LeviUniAcc**
@1Maxnet1 @EhrenbruderRoschtbraten

### CODE REVIEW

**Review by EhrenbruderRoschtbraten — APPROVED** [review]

File: paper.tex [hunk 1]
```diff
@@ -69,6 +69,8 @@ \subsection{Detector models}
 
 \subsubsection{Image detector models}
 
+The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, pretrained image detection model. For finetuneing the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```
Comment (RaphaW) [inline paper.tex]:
```suggestion
The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, pretrained image detection model. For finetuning the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```

Comment (RaphaW) [reply paper.tex]: Why not a bibliography entry ?

Comment (LeviUniAcc) [reply paper.tex]: couse i didnt know how, will fix that later

Comment (RaphaW) [reply paper.tex]: I wouldn't resolve this conversation if it is not already fixed.
Otherwise it could happen that you forget it

**Review by RaphaW — APPROVED** [review]
If you chnage the citation here as well, then it looks good

File: paper.tex [hunk 2]
```diff
@@ -69,6 +69,8 @@ \subsection{Detector models}
 
 \subsubsection{Image detector models}
 
+The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, pretrained image detection model. For finetuning the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```
Comment (1Maxnet1) [inline paper.tex]: I would remove the first pretrained, as the model is not a foundation model and therefore I would not call it pretrained.
```suggestion
The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, image detection model. For fine-tuning the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```

**Review by 1Maxnet1 — APPROVED** [review]
One suggestion and fixing the citation is necessary, but otherwise fine

**Pull request merged**